config/routes.rb

```diff
@@ -53,4 +53,9 @@
   #     # (app/controllers/admin/products_controller.rb)
   #     resources :products
   #   end
+
+  namespace :api do
+    resources :category
```

Comment:
The convention is to use the plural name of the collection. See some of the examples in the comments in this file, above. The controllers should be renamed to use the plural as well, e.g. `CategoriesController`.